nautobot_golden_config/nornir_plays/config_backup.py

```diff
@@ -54,39 +55,41 @@ def run_backup(  # pylint: disable=too-many-arguments
     backup_obj.backup_last_attempt_date = task.host.defaults.data["now"]
     backup_obj.save()
 
-    backup_path_template_obj = render_jinja_template(obj, logger, global_settings.backup_path_template)
-    backup_file = os.path.join(backup_root_folder, backup_path_template_obj)
-
-    if global_settings.backup_test_connectivity is not False:
-        task.run(
+    for backup_repo in backup_repos:
```

Comment:
I am not following why this would be in a for loop? If there is 3 backup repos, this seems to me like it would run 3 backups or always return after the first one? 

I kinda presumed there would be a `find_expected_backup_path()` function, and then run everything else the same.